### ISSUE TITLE
Frozen Column Header Scroll Issue

### ISSUE THREAD

**Issue by BusyWeb** [JLynch7/SlickGrid#6]
I really like your implementation of slickgrid, Frozen Column, Row.
After some study, I decided to use your code on my project and seems work fine.
However, I found an issue  that  the Header columns scrolling is stopping at the certain points (but other data rows are scrolling fine >> horizontally).
If I remove option FrozenColumns, header column scrolling is fine to the end of the columns.
Would you check? 
I tried to find what ? but could not find the solution from code yet.
Thanks

**Comment by JLynch7** [comment]
I'm having trouble reproducing this issue.  Can you modify one of the examples and cause the same behavior?

**Comment by BusyWeb** [comment]
Hi, there.
After some struggling with code, I found a place that solved my problem.
At the code line 270, 271 on slikcgrid.js   >>>   Append the columnn containers to the headers
            $headerL = $("<div class='slick-header-columns slick-header-columns-left' style='width:10000px; left:-1000px' />").appendTo($headerScrollerL);
            $headerR = $("<div class='slick-header-columns slick-header-columns-right' style='width:10000px; left:-1000px' />").appendTo($headerScrollerR);

I updated the width to 20000 (double size), and then worked well without the stopping header scrolling.
My project's columns counts are about 80, and thought that there are some limits on the size.
Tried to find and update the "Width" props, ha ha ha .. my guess was fortunately right.. ^^;

Thanks for your Frozen column implementation, it is really good solution for my working project and really like it.
Thanks again.
Jung Yi

-----Original Message-----
